packages/api/src/memberContext.ts

```diff
@@ -312,12 +312,18 @@ export class MemberContext implements MemberContext {
     const lookAheadDate = new Date(startDate.getTime() + daysToLookAhead * ONE_DAY_IN_MILLISECONDS)
 
     const subscriptionsPaidUntil: Subscription[] = []
+    // max batches is a security feature, which prevents in case of an auto-renew bug too many people are going to be charged unintentionally.
+    const maxSubscriptionBatch = parseInt(process.env.MAX_AUTO_RENEW_SUBSCRIPTION_BATCH || 'false')
+    // max batch size is 100 given by https://github.com/wepublish/wepublish/blob/master/packages/api-db-mongodb/src/db/defaults.ts#L3
+    const batchSize =
+      !isNaN(maxSubscriptionBatch) && maxSubscriptionBatch < 100 ? maxSubscriptionBatch : 100
```

Comment:
could use `Math.min(maxSubscriptionBatch, 100)` here, maybe cleaner?

Reply:
@michael-scheurer true but probably not as efficient ;-)

Reply:
Performance wise? They are equal :D

Reply:
what happens if maxSubscriptionBatch is NaN inside Math.min()?

Reply:
Returns NaN, but it's more for the manual checking afterwards.

Im used to write/read Math.min for this so I am a bit biased as its easier to read for me.